### PR TITLE
Add "All Media" option to global menu

### PR DESCRIPTION
**Pull request by adbenitez**
add a global profile to browse all media/documents/attachments, games etc. really useful to find attachments when user can't remember in which chat was sent, also useful for quickly reviewing and removing big attachments to save space

(note: most stuff was already in place to support this, thanks to that fellow collaborator from the past!)

### CODE REVIEW

**Comment by adbenitez** [comment]
btw, the "help" menu option could be removed since it is already available in the settings

File: src/org/thoughtcrime/securesms/ProfileActivity.java [hunk 1]
```diff
@@ -97,17 +97,21 @@ protected void onCreate(Bundle bundle, boolean ready) {
...
-    Toolbar parent = (Toolbar) supportActionBar.getCustomView().getParent();
-    parent.setPadding(0,0,0,0);
-    parent.setContentInsetsAbsolute(0,0);
-
-    titleView = (ConversationTitleView) supportActionBar.getCustomView();
-    titleView.setOnBackClickedListener(view -> onBackPressed());
-    titleView.setOnClickListener(view -> onEnlargeAvatar());
+    if (isGlobalProfile()) {
+      supportActionBar.setDisplayHomeAsUpEnabled(true);
+      supportActionBar.setHomeActionContentDescription(getString(R.string.back));
```
Comment (r10s) [inline src/org/thoughtcrime/securesms/ProfileActivity.java:102]: is that really needed? i would expect that the default text just reads "back". at least we are not doing that at all the other placed where we set the back-button.

maybe better remove this line here - and if we really think, it is important, do a separate pr and set that at all places.

**Review by r10s — APPROVED** [review]
nice! 

such an impact with such few lines of changes - interestingly,  the string "All Media" was even already there :)

only a few nitpicks, see comments below. for the "help" button - yip, i also think that can be removed, "help" is pretty common in the settings on android, but we can also do that in a subsequent pr.

File: src/org/thoughtcrime/securesms/ProfileActivity.java (hunk 1 above)
Comment (adbenitez) [reply src/org/thoughtcrime/securesms/ProfileActivity.java:102]: it is needed because android sets "navigate up" as content description, the other back buttons are part of our custom view
